sample.env

```diff
@@ -6,4 +6,6 @@ FIREBASE_CLIENT_EMAIL=INSERT_HERE
 FIREBASE_WEB_API_KEY=INSERT_HERE
 FIREBASE_SERVICE_ACCOUNT_ID=INSERT_HERE
 GOOGLE_API_KEY=your_key
-GEOCODING_KEY=INSERT_HERE
\ No newline at end of file
+GEOCODING_KEY=INSERT_HERE
+S3_ACCESS_KEY_ID=INSERT_HERE
+S3_SECRET_ACCESS_KEY=INSERT_HERE
```

Comment:
@ubclaunchpad/foodies I've updated the env file with new environment variables https://drive.google.com/drive/u/0/folders/1YhX35Vk8I0_PECrk6I2btsFPdd8BC-ow. Please take a look as I put a comment how to get your S3 env variables